### PR TITLE
ci: add a test for release steps

### LINKED ISSUE #87: Release steps should be testable
At least we need to ensure that all steps are OK without actual publishing to npm or gh-pages.

### PR DESCRIPTION
**Pull request by romanrostislavovich**
Added tests for release steps.

Add new gulp commands:
- `npm:itself` - install all local packages from `tgz` files in root directory
- `npm:pack` - run `npm pack` command

Closes #87

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ gulp.task('lint', gulp.series('lint:all'));
  */
 gulp.task('packages:reinstall', gulp.series(
     'clean:node',
-    'packages:install'
+    'npm:install'
```
Comment (xferra) [inline gulpfile.js]: Again npm install?

Comment (xferra) [reply gulpfile.js]: Pwease staph

File: script/gulp/task.npm.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    .pipe(gulpExec(`npm pack`))
+    .pipe(gulpExec.reporter());
+
+// Automatically install all npm packages
+const install = () => gulp
+    .src(config.root.package)
+    .pipe(gulpInstall());
+
+// Install all local packages from `tgz` files in root directory
+const itself = (done) => gulp
```
Comment (xferra) [inline script/gulp/task.npm.js]: Well, we can find a better name for that.

File: script/gulp/task.npm.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    .src('./*.tgz')
```
Comment (xferra) [inline script/gulp/task.npm.js]: Const, mask. Any idea about a better place for that?)

File: script/gulp/task.npm.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    .pipe(gulpExec('npm install <%= file.path %> --force'))
```
Comment (xferra) [inline script/gulp/task.npm.js]: Well, is it a new template engine for strings?

Comment (romanrostislavovich) [reply script/gulp/task.npm.js]: It's `lodash` template :)

File: script/gulp/task.npm.js (hunk 3 above)
Comment (romanrostislavovich) [reply script/gulp/task.npm.js]: done

File: script/gulp/task.npm.js (hunk 2 above)
Comment (romanrostislavovich) [reply script/gulp/task.npm.js]: done

File: gulpfile.js (hunk 1 above)
Comment (romanrostislavovich) [reply gulpfile.js]: done

**Review by xferra — APPROVED** [review]
+1

**Pull request merged**